src/structures/Message.js

```diff
@@ -422,10 +422,16 @@ class Message extends Base {
     );
   }
 
+  /**
+   * The Message this crosspost/reply/pin add references, if cached
+   * @type {Message}
+   * @readonly
+   */
```

Comment:
```suggestion
  /**
   * The Message this crosspost/reply/pin add references, if cached
   * @type {?Message}
   * @readonly
   */
```
`referencedMessage` can be `null`.

Reply:
Thanks!